frontend/src/domains/DomainCard.js

```diff
@@ -127,13 +188,40 @@ export function DomainCard({ url, status, hasDMARCReport, tags, ...rest }) {
             </Button>
           )}
         </Stack>
-        <ScanDomainButton domainUrl={url} ml={4} />
+        <Stack ml={4}>
+          <ScanDomainButton domainUrl={url} />
+          <ABTestingWrapper insiderVariantName="B">
+            <ABTestVariant name="B">
+              {isLoggedIn() &&
+                (location.pathname.match('my-tracker') ? (
+                  <IconButton
+                    onClick={async () => {
+                      await unfavouriteDomain({ variables: { domainId: id } })
+                    }}
+                    variant="primary"
+                    aria-label={`unfavourite ${url}`}
+                    icon={<StarIcon color="moderate" />}
+                  />
+                ) : (
+                  <IconButton
+                    onClick={async () => {
+                      await favouriteDomain({ variables: { domainId: id } })
+                    }}
+                    variant="primary"
+                    aria-label={`favourite ${url}`}
+                    icon={<StarIcon />}
+                  />
+                ))}
+            </ABTestVariant>
+          </ABTestingWrapper>
+        </Stack>
```

Comment:
It would be clean if we could show the `favourite` status on the normal "Domains" page, while also allowing the user to unfav on the "Domains" page. How feasible is that?

Reply:
Yeah, I agree. I'm thinking we add that in later, and merge in what we have now.